staging.Dockerfile

```diff
@@ -0,0 +1,74 @@
+# This Docker file tends to be used only
+# for `development` and `test` stage
```

Comment:
This comment seems wrong now, in the context of this file.

Reply:
Agreed, will clean up file and explain some choices / logics in comments.